<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ADUserInfoWidgetUI</class>
 <widget class="QWidget" name="ADUserInfoWidgetUI">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>597</width>
    <height>575</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>AD User Info</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_4">
   <item row="0" column="0">
    <widget class="QGroupBox" name="groupBoxAccount">
     <property name="title">
      <string>Account</string>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="6" column="2" colspan="2">
       <widget class="QCheckBox" name="checkBoxUnlockAccount">
        <property name="text">
         <string>Unlock Account</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="labelSAMAccountName">
        <property name="text">
         <string>Account Name:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLineEdit" name="lineEditSAMAccount">
        <property name="toolTip">
         <string>SAM Account Name</string>
        </property>
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QLabel" name="labelDisplayName">
        <property name="text">
         <string>Display Name:</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item row="0" column="3">
       <widget class="QLineEdit" name="lineEditDisplayName">
        <property name="toolTip">
         <string>Display Name</string>
        </property>
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="labelDescription">
        <property name="text">
         <string>Description:</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1" colspan="3">
       <widget class="QLineEdit" name="lineEditDescription">
        <property name="toolTip">
         <string>Description</string>
        </property>
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="2" column="1" colspan="3">
       <widget class="QLineEdit" name="lineEditUserWorkstations">
        <property name="toolTip">
         <string>Workstations User Can Logon To</string>
        </property>
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="labelUserWorkstations">
        <property name="text">
         <string>Workstations:</string>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="labelTelephone">
        <property name="text">
         <string>Telephone:</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1" colspan="3">
       <widget class="QLineEdit" name="lineEditTelephone">
        <property name="toolTip">
         <string>Telephone Number</string>
        </property>
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="5" column="0">
       <widget class="QLabel" name="labelOU">
        <property name="text">
         <string>OU:</string>
        </property>
       </widget>
      </item>
      <item row="5" column="1">
       <widget class="QComboBox" name="comboBoxOU">
        <property name="enabled">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item row="6" column="0" colspan="2">
       <widget class="QCheckBox" name="checkBoxAccountDisabled">
        <property name="text">
         <string>Account Disabled</string>
        </property>
       </widget>
      </item>
      <item row="4" column="0">
       <widget class="QLabel" name="labelDepartment">
        <property name="text">
         <string>Department:</string>
        </property>
       </widget>
      </item>
      <item row="4" column="1">
       <widget class="QLineEdit" name="lineEditDepartment">
        <property name="toolTip">
         <string>Department</string>
        </property>
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="4" column="2">
       <widget class="QLabel" name="labelTitle">
        <property name="text">
         <string>Title:</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item row="4" column="3">
       <widget class="QLineEdit" name="lineEditTitle">
        <property name="toolTip">
         <string>Title</string>
        </property>
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QGroupBox" name="groupBoxID">
     <property name="title">
      <string>ID</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="0" column="0">
       <widget class="QLabel" name="label">
        <property name="text">
         <string>GUID:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLineEdit" name="lineEditGUID">
        <property name="toolTip">
         <string>GUID</string>
        </property>
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="labelSID">
        <property name="text">
         <string>SID:</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLineEdit" name="lineEditSID">
        <property name="toolTip">
         <string>SID</string>
        </property>
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QGroupBox" name="groupBoxUseLocalComputer">
     <property name="title">
      <string>Use Local Computer Resources</string>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
     <property name="checked">
      <bool>false</bool>
     </property>
     <layout class="QGridLayout" name="gridLayout_3">
      <item row="0" column="1" colspan="3">
       <widget class="QCheckBox" name="checkBoxLogonOnly">
        <property name="text">
         <string>Allowing to logon to this computer only</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <spacer name="horizontalSpacer_3">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="1" column="0" colspan="4">
       <widget class="QGroupBox" name="groupBoxLocalGroup">
        <property name="title">
         <string>Local User Group</string>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QCheckBox" name="checkBoxPowerUsers">
           <property name="text">
            <string>Power Users</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="checkBoxRDusers">
           <property name="text">
            <string>Remote Desktop Users</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="checkBoxAdministrators">
           <property name="text">
            <string>Administrators</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="4" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QPushButton" name="pushButtonEdit">
       <property name="text">
        <string>&amp;Edit</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="pushButtonClose">
       <property name="toolTip">
        <string>Close</string>
       </property>
       <property name="text">
        <string>&amp;Close</string>
       </property>
       <property name="default">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="1" column="0">
    <widget class="QGroupBox" name="groupBoxPassword">
     <property name="title">
      <string>Password</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_5">
      <item row="1" column="0">
       <widget class="QCheckBox" name="checkBoxUserMustChangePassword">
        <property name="text">
         <string>User must change password at next logon</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QCheckBox" name="checkBoxUserCannotChangePassword">
        <property name="text">
         <string>User can not change password</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QFrame" name="framePassword">
        <property name="frameShape">
         <enum>QFrame::NoFrame</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <property name="leftMargin">
          <number>1</number>
         </property>
         <property name="topMargin">
          <number>1</number>
         </property>
         <property name="rightMargin">
          <number>1</number>
         </property>
         <property name="bottomMargin">
          <number>1</number>
         </property>
         <item>
          <widget class="QLabel" name="labelPassword">
           <property name="text">
            <string>Password:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="lineEditPassword">
           <property name="echoMode">
            <enum>QLineEdit::Password</enum>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="labelConfirmPassword">
           <property name="text">
            <string>Confirm Password:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="lineEditConfirmPassword">
           <property name="echoMode">
            <enum>QLineEdit::Password</enum>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QCheckBox" name="checkBoxPasswordNeverExpires">
        <property name="text">
         <string>Password never expires</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>lineEditSAMAccount</tabstop>
  <tabstop>lineEditDisplayName</tabstop>
  <tabstop>lineEditDescription</tabstop>
  <tabstop>lineEditUserWorkstations</tabstop>
  <tabstop>lineEditTelephone</tabstop>
  <tabstop>lineEditDepartment</tabstop>
  <tabstop>lineEditTitle</tabstop>
  <tabstop>comboBoxOU</tabstop>
  <tabstop>checkBoxAccountDisabled</tabstop>
  <tabstop>checkBoxUnlockAccount</tabstop>
  <tabstop>lineEditPassword</tabstop>
  <tabstop>lineEditConfirmPassword</tabstop>
  <tabstop>checkBoxUserMustChangePassword</tabstop>
  <tabstop>checkBoxUserCannotChangePassword</tabstop>
  <tabstop>checkBoxPasswordNeverExpires</tabstop>
  <tabstop>lineEditGUID</tabstop>
  <tabstop>lineEditSID</tabstop>
  <tabstop>groupBoxUseLocalComputer</tabstop>
  <tabstop>checkBoxLogonOnly</tabstop>
  <tabstop>checkBoxPowerUsers</tabstop>
  <tabstop>checkBoxRDusers</tabstop>
  <tabstop>checkBoxAdministrators</tabstop>
  <tabstop>pushButtonEdit</tabstop>
  <tabstop>pushButtonClose</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
